README.adoc

```diff
@@ -409,7 +409,7 @@ InventoryManager.java
 include::finish/inventory/src/main/java/io/openliberty/guides/inventory/InventoryManager.java[]
 ----
 
-Annotate the [hotspot=addMethod file=0]`add()` and [hotspot=listMethod file=0]`list()` methods with the [hotspot=addWithSpan hotspot=listWithSpan file=0]`@WithSpan` annotation. You can annotate the [hotspot=spanAttribute file=0]`host` parameter with the [hotspot=spanAttribute file=0]`@SpanAttribute` annotation with a customized value to indicate that it is part of the trace.
+Annotate the [hotspot=addMethod file=0]`add()` and [hotspot=listMethod file=0]`list()` methods with the [hotspot=addWithSpan hotspot=listWithSpan file=0]`@WithSpan` annotation. Then, the OpenTelemetry instrumentation will provide a new span for each method. This will allow you to collect and trace the spans across different services. Additionally, you can annotate the [hotspot=spanAttribute file=0]`host` parameter with the [hotspot=spanAttribute file=0]`@SpanAttribute` annotation with a customized value to indicate that it is part of the trace.
```

Comment:
```suggestion
The [hotspot=addMethod file=0]`add()` and [hotspot=listMethod file=0]`list()` methods are annotated with the [hotspot=addWithSpan hotspot=listWithSpan file=0]`@WithSpan` annotation. The OpenTelemetry instrumentation provides a new span for each method. You can now collect and trace the spans across different services. The [hotspot=spanAttribute file=0]`host` parameter is also annotated with the [hotspot=spanAttribute file=0]`@SpanAttribute` annotation with a customized value to indicate that it is part of the trace.
```
This whole bit after the initial instruction (Replace the xx class:) is meant to be describing what doing that action has achieved. I've edited this one to show what I mean - happy to chat more if it's not clear. Also, could you check that the last sentence is accurate - I assumed that where it said "Additionally, you can annotate...." it means that that's been done in the updated class file but it may mean you can optionally go on to do that yourself (which seems less helpful tbh).